internal/gribi/gribi.go

```diff
@@ -218,6 +218,14 @@ func (c *Client) AddNH(t testing.TB, nhIndex uint64, address, instance string, e
 			NH = NH.WithNextHopNetworkInstance(opt.VrfName)
 		}
 		c.fluentC.Modify().AddEntry(t, NH)
+	case "VRFOnly":
+		NH := fluent.NextHopEntry().
```

Comment:
lower case, because you are not planning to export this variable?

Reply:
Addressed the comment